celery_migration_app.py

```diff
@@ -20,7 +22,7 @@
     'accept_content': ['json']})
 
 
-@app.task
+@app.task(autoretry_for=(RequestException,HttpError), retry_backoff=True)
```

Comment:
Tried to run with concurrency=4, ran into quota issues, so implementing exponential backoff.

"Quota exceeded for quota metric 'photoslibrary.googleapis.com/write_requests' and limit 'WritesPerMinutePerUser' of service 'photoslibrary.googleapis.com'"